mods/ra2/rules/allied-structures.yaml

```
@@ -57,10 +57,12 @@ gacnst:
 	GrantConditionOnPowerState@LOWPOWER:
 		Condition: lowpower
 		ValidPowerStates: Low, Critical
+	WithIdleOverlay@fans@LOWPOWER:
+		Sequence: idle-fans-low-power
+		RequiresCondition: !build-incomplete
```

I'm not 100% sure, but this seems like it would play with (under) the other idle animation if not on low power, so I suggest 
```suggestion
		RequiresCondition: !build-incomplete && lowpower
```
to be safe.

---

Yea, I originally had that condition but removed it thinking it is a bit redundant but I guess it makes more sense to add a little safeguard just in case